docs/helium-tokens/datacredit.mdx

```diff
@@ -130,8 +130,8 @@ acquiring Data Credits.
 
 ### Sphere Data Credit Portal
 
-The [Data Credit Portal](/data-credit-portal) built by [Sphere](https://sphere.engineer) allows
-network users to acquire Data Credits without directly holding cryptocurrency. The tool is designed
-for allocating Data Credits for network users like [LNS operators](/iot/open-lns) or mobile
+The [Data Credit Portal](/data-credit-portal) built by [Sphere](https://helium.sphere.engineer)
```

Comment:
This link is meant to be sphere.engineer.

Reply:
fix included in #1363